RELEASE_NOTES

```diff
@@ -1,3 +1,4 @@
+- Refactor: Migrate the BibliographicReferences to BibliographicReference [#qubIT-Fenix-1595]
```

Comment:
I would suggest something like this: _Refactor: Prepare migration of bibliographic references from value types to domain entities_